src/content/developers/docs/smart-contracts/formal-verification/index.md

```diff
@@ -0,0 +1,272 @@
+
+**Act**: _*Act allows specification of storage updates, pre/post conditions and contract invariants. Its tool suite also has proof backends able to prove many properties via Coq, SMT solvers, or hevm.*_
+
+- [GitHub](https://github.com/ethereum/act)
+- [Documentation](https://ethereum.github.io/act/)
+
+**Scribble** - _*Scribble transforms code annotations in the Scribble specification language into concrete assertions that check the specification.*_
+
+- [Documentation](https://docs.scribble.codes/)
+
+**Dafny** - _*Dafny is a verification-ready programming language that relies on high-level annotations to reason about and prove correctness of code.*_
+
+- [GitHub](https://github.com/dafny-lang/dafny)
+
+### Program verifiers for checking correctness {#program-verifiers}
+
+**Solidity SMTChecker** - *_Solidity’s SMTChecker is a built-in model checker based on SMT (Satisfiability Modulo Theories) and Horn solving. It confirms if a contract’s source code matches specifications during compilation and statically checks for violations of safety properties._*
+
+- [GitHub](https://github.com/ethereum/solidity)
+
+**solc-verify** - *_solc-verify is an extended version of the Solidity compiler that can perform automated formal verification on Solidity code using annotations and modular program verification._*
+
+- [GitHub](https://github.com/SRI-CSL/solidity)
+
+**hevm** - *_hevm is a symbolic execution engine and equivalence checker for EVM bytecode._*
+
+- [GitHub](https://github.com/dapphub/dapptools/tree/master/src/hevm)
+
+**KEVM**: _*KEVM is a formal semantics of the Ethereum Virtual Machine (EVM) written in the K framework. KEVM is executable and can prove certain property-related assertions using reachability logic.*_
+
+- [GitHub](https://github.com/runtimeverification/evm-semantics)
+- [Documentation](https://jellopaper.org/)
+
+### Symbolic execution-based tools for detecting vulnerable patterns in smart contracts {#symbolic-execution-tools}
+
+**Manticore** - *_A tool for analyzing EVM bytecode analysis tool based on symbolic execution_.*
+
+- [GitHub](https://github.com/trailofbits/manticore)
+- [Documentation](https://github.com/trailofbits/manticore/wiki)
+
+**Securify** - _*A security analyzer for analyzing Solidity source code in Ethereum smart contracts and detecting property violations.*_
+
+- [GitHub](https://github.com/eth-sri/securify2)
+
+**Oyente** - *_Static analysis tool for analyzing vulnerabilities in EVM bytecode with symbolic execution._*
+
+- [GitHub](https://github.com/melonproject/oyente)
+
+**Echidna** - *_Echidna is a fast contract fuzzer for detecting vulnerabilities in smart contracts through property-based testing._*
+
+- [GitHub](https://github.com/crytic/echidna/)
+
+**Harvey** - *_Harvey is an automated fuzzing tool useful for detecting property violations in smart contract code._*
+
+- [Website](https://consensys.net/diligence/fuzzing/)
+
```

Comment:
Echidna and Harvey are fuzzers, so maybe it'd make sense to have a fuzzers section even though this is an FV page? Or maybe move that to the testing page actually?

Reply:
That makes sense. I took them out now.